app/models/transport_requisition.rb

```diff
@@ -15,11 +15,22 @@
 #
 
 class TransportRequisition < ApplicationRecord
+  enum status: [:open, :closed]
   has_many :transport_requisition_items
   belongs_to :operation
 
   def region
```

Comment:
@nathnael this will penalize performance greatly. as it will have to go to the db to fetch region information - not good.